### DISCUSSION TITLE
PRs

### DISCUSSION THREAD

**Discussion by kerizane** [Tuxemon/Tuxemon#2351] [General]
Hey Everyone,

Sorry the PRs haven't been merged for a while, I've been very busy at work. I've started merging them, but still have a lot to do. I will probably chip away at them over a week or two. Please let me know if there's any I should prioritise.

**Comment by kerizane** [comment]
@JaskRendix

**Reply by JaskRendix** [reply]
@kerizane no problem, happy to see you back, at the moment I don't see anything that should be urgently merged .

**Comment by Sanglorian** [comment]
Thanks so much, @kerizane !

**Comment by ultidonki** [comment]
Yeah, thanks a lot for merging those PR's! I think the one I wanted most was already merged, #2317 - this fixes a bunch of bugs I noticed in the game, so that's great it's been merged already. :smile:
